CLA_SIGNERS.yaml

```diff
@@ -88,6 +88,9 @@ people:
   - name: Steffen Winandy
     email: stewin19@gmail.com
     github: StefWin
+  - name: Vladimir Bogomolov
+    email: vladimirbogomoloff@gmail.com
```

Comment:
Is the email correct? On his [github profile](https://github.com//VladimirBogomolov) it looks like there's a dot between his first and last name.

Reply:
Commits signed with `vladimirbogomoloff@gmail.com`. Not sure but maybe he has two emails.

Reply:
@VladimirBogomolov

Reply:
Email matches his commit and signing. lgtm

Reply:
I totally forgot, but in gmail the periods don't actually matter. [Link](https://gmail.googleblog.com/2008/03/2-hidden-ways-to-get-more-from-your.html)

Reply:
It was actually filed in the paperwork as `VladimirBogomoloff@gmail.com` but I went with all lowercase to match the commit.

@VladimirBogomolov please chime in if the values here don't match your [signed commits](https://github.com/JanusGraph/janusgraph/blob/master/CONTRIBUTING.md#configure-your-repo-to-match-the-cla). The project's CLA checker is very picky ;)

Reply:
The CLA checker is case-insensitive for emails, so if you want to add it as TitleCase, it will still work. It doesn't handle the special case of periods being optional in Gmail, as @chupman pointed out.

Reply:
I just see that signed commits from @VladimirBogomolov are not recognized by GitHub as commits from @VladimirBogomolov .

@VladimirBogomolov , I am not sure but possibly you linked GitHub account to  `vladimir.bogomoloff@gmail.com`. Maybe GitHub will recognize your commits if you link `vladimirbogomoloff@gmail.com` to your account?

Reply:
@pluradj note that you can add 2 entries for a single individual: one with the email with the dot, and one without (similarly to how it was added for folks who have 2 email addresses for CLI use vs. web UI use), that will cover both cases.

Reply:
My apologies.
It was a mistake in my git config, user.email was without dot.
I have fixed config and commitmessages.

Sorry.